dev-python/pydotplus/pydotplus-2.0.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Improved version of the old pydot project"
+HOMEPAGE="http://pydotplus.readthedocs.org/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```

Comment:
`IUSE="test"` is orthogonal to running tests. `IUSE="test"` is used for pulling in dependencies needed for running tests, not actually for enabling/disabling tests (that is done by `FEATURES="test"`, unless you use paludis)